### DISCUSSION TITLE
How to access PostgresDB on local setup?

### DISCUSSION THREAD

**Discussion by Haberkamp** [supabase/supabase#7335] [Questions] [answered]
Hello, 

is it possible to connect to the local supabase Postgres DB? There are two things I want to do: 1. Connect my PHPStorm IDE with the database and second more important I want to connect Prisma to the database.

Thank you in advance!

**Comment by silentworks** [comment, marked as answer]
You should be able to connect to the Postgres DB using the following credentials

host: localhost
port: 5432
database: postgres
user: posttgres
password: this would be something you set in your .env file if you are using the supabase docker instance
